pom.xml

```diff
@@ -3036,7 +3036,16 @@
       <properties>
         <maven.compiler.release>${releaseTarget}</maven.compiler.release>
         <!-- TODO: replicate logic for windows support -->
-        <argLine>--add-opens=java.base/jdk.internal.ref=ALL-UNNAMED ${hbase-surefire.argLine}</argLine>
+        <argLine>-Dio.netty.tryReflectionSetAccessible=true
+          --illegal-access=permit
+          --add-modules jdk.unsupported
+          --add-opens java.base/java.nio=ALL-UNNAMED
+          --add-opens java.base/sun.nio.ch=ALL-UNNAMED
+          --add-opens java.base/java.lang=ALL-UNNAMED
+          --add-opens=java.base/jdk.internal.ref=ALL-UNNAMED
```

Comment:
This one `add-opens` line uses an `=` to specify its argument, the rest don't. Mine making them all uniform? Same thing in `bin/hbase`.

Reply:
Ah, copy-paste error. Thanks for pointing this out.

Let me change it.